doc/sfp-cmis/Interface-Link-bring-up-sequence.md

```diff
@@ -0,0 +1,160 @@
+# Feature Name
+Deterministic Approach for Interface Link bring-up sequence
+
+# High Level Design Document
+#### Rev 0.3
+
+# Table of Contents
+  * [List of Tables](#list-of-tables)
+  * [Revision](#revision)
+  * [About This Manual](#about-this-manual)
+  * [Abbreviation](#abbreviation)
+  * [References](#references)
+  * [Problem Definition](#problem-definition)
+  * [Background](#background)
+  * [Objective](#objective)
+  * [Proposal](#proposal)
+  * [Proposed Work-Flows](#proposed-work-flows)
+
+# List of Tables
+  * [Table 1: Definitions](#table-1-definitions)
+  * [Table 2: References](#table-2-references)
+
+# Revision
+| Rev |     Date    |       Author                       | Change Description                |
+|:---:|:-----------:|:----------------------------------:|-----------------------------------|
+| 0.1 | 08/16/2021  | Shyam Kumar                        | Initial version                       
+| 0.2 | 12/13/2021  | Shyam Kumar,  Jaganathan Anbalagan | Added uses-cases, workflows  
+| 0.3 | 01/19/2022  | Shyam Kumar,  Jaganathan Anbalagan | Addressed review-comments    
+| 0.4 | 01/26/2022  | Shyam Kumar,  Jaganathan Anbalagan | Addressed further review-comments 
+
+
+# About this Manual
+This is a high-level design document describing the need to have determinstic approach for
+Interface link bring-up sequence and workflows for use-cases around it 
+
+# Abbreviation
+
+# Table 1: Definitions
+| **Term**       | **Definition**                                   |
+| -------------- | ------------------------------------------------ |
+| pmon           | Platform Monitoring Service                      |
+| xcvr           | Transceiver                                      |
+| xcvrd          | Transceiver Daemon                               |
+| CMIS           | Common Management Interface Specification        |
+| gbsyncd        | Gearbox (External PHY) docker container          |
+| DPInit         | Data-Path Initialization                         |
+| QSFP-DD        | QSFP-Double Density (i.e. 400G) optical module   |
+
+# References
+
+# Table 2 References
+
+| **Document**                                            | **Location**  |
+|---------------------------------------------------------|---------------|
+| CMIS v5 | [CMIS5p0.pdf](http://www.qsfp-dd.com/wp-content/uploads/2021/05/CMIS5p0.pdf) |
```

Comment:
Also list CMIS4 spec since it's referenced below?

Reply:
Will add the link for CMIS 4.0